PKGBUILD

```diff
@@ -6,14 +6,13 @@ pkgrel=1
 pkgdesc="Паренная репа нахуй"
 arch=('any')
 url="https://github.com/Zayac-The-Engineer/${pkgname}"
-license=('MIT')
+license=('GPL')
 depends=()
-makedepends=('git')
+makedepends=('git' 'cmake')
 source=("${pkgname}-${pkgver}.tar.gz::https://github.com/Zayac-The-Engineer/${pkgname}/archive/${pkgver}.tar.gz")
 md5sums=('SKIP')
 
 package() {
 	cd "$srcdir/$pkgname-$pkgver"
 	make DESTDIR="$pkgdir" install
-	install -Dm644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
```

Comment:
Why remove this?

Reply:
See https://wiki.archlinux.org/index.php/PKGBUILD#license

> The license under which the software is distributed. The licenses package (a dependency of the base meta package) contains many commonly used licenses, which are installed under /usr/share/licenses/common/. If a package is licensed under one of these licenses, the value should be set to the directory name, e.g. license=('GPL').

Reply:
I mean, why remove the installation of the license?

Reply:
Because the GNU GPL it's already in the _licenses_ package in the /usr/share/licenses/common/GPL(2, 3). You have to include the license in the package only if you're using a license which is not in the _licenses_ package